packages/block-library/src/image/index.php

```diff
@@ -272,12 +273,14 @@ function block_core_image_print_lightbox_overlay() {
 		<div
 			class="wp-lightbox-overlay zoom"
 			data-wp-interactive="core/image"
+			data-wp-router-region='{ "id": "core/body", "attachTo": "body" }'
```

Comment:
This should be more explicit.

```suggestion
			data-wp-router-region='{ "id": "core/image-overlay", "attachTo": "body" }'
```